components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/GraphBasedSequenceHandler.java

```diff
@@ -486,6 +490,23 @@ private boolean callExternalSystem(HttpServletRequest request, HttpServletRespon
         return false;
     }
 
+    private int getLongWaitTimeout() {
+
+        String waitTimeoutPropValue = IdentityUtil.getProperty(ADAPTIVE_AUTH_LONG_WAIT_TIMEOUT);
+        int waitTimeout = 10000;
+        try {
+            if (waitTimeoutPropValue != null) {
+                waitTimeout = Integer.parseInt(waitTimeoutPropValue);
+            }
+        } catch (NumberFormatException e) {
+            //ignore. Default value will be used.
+            if (log.isDebugEnabled()) {
+                log.debug("Error while reading the wait timeout. ", e);
```

Comment:
Shall we make this a warn log and also mention that the default value 10000 will be used